docs/main/updating/4-0.md

```diff
@@ -41,6 +41,12 @@ post_install do |installer|
 end
 ```
 
+Additionally, update the `podspec` file at the root of the project:
```

Comment:
You have put this in the app update guide, not in the plugin update guide

Reply:
Oops! fixed